packages/patternfly-4/react-core/src/components/ApplicationLauncher/ApplicationLauncher.docs.js

```diff
@@ -0,0 +1,15 @@
+import { ApplicationLauncher } from '@patternfly/react-core';
+import SimpleApplicationLauncher from './examples/SimpleApplicationLauncher';
+import TooltipApplicationLauncher from './examples/TooltipApplicationLauncher';
+
+export default {
+  title: 'Dropdown',
+  components: {
```

Comment:
Also add something like
```
enumValues: {
    'Object.values(AlertVariant)': Object.values(DropdownPosition)
  },
```
so that docs lists them

Reply:
Changed to Application Launcher

Reply:
Changed to object.values here and in drop down code as well.